app/build.gradle.kts

```diff
@@ -53,6 +53,7 @@ android {
 
     kotlinOptions {
         jvmTarget = "1.8"
+        useIR = true
```

Comment:
I don't think it's necessary now but it's not doing any harm so I'll just leave it